internal/wasm/jit/compiler.go

```diff
@@ -337,4 +337,24 @@ type compiler interface {
 	// compileSignExtend64From32 adds instruction to sign-extends the first 32-bits of 64-bit in as signed 64-bit int.
 	// See wasm.OpcodeI64Extend32S
 	compileSignExtend64From32() error
+	// compileMemoryInit adds instructions to perform operations corresponding to the memory.init instruction in
+	// the bulk-memory-operations proposal.
+	//
+	// https://github.com/WebAssembly/spec/blob/main/proposals/bulk-memory-operations/Overview.md
```

Comment:
use the spec 2.0 link to memory.init (here and below)? https://www.w3.org/TR/2022/WD-wasm-core-2-20220419/ ...